python/py-gitfs/Portfile

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-gitfs
+github.setup        presslabs ${python.rootname} 0.5.2
+revision            0
+
+categories-append   devel
+platforms           darwin
+license             Apache-2
+maintainers         nomaintainer
+
+description         a FUSE file system that fully integrates with git
+long_description    ${python.rootname} is {*}${description}. You can mount a\
+                    remote repository’s branch locally, and any subsequent\
+                    changes made to the files will be automatically committed\
+                    to the remote.
+
+homepage            https://www.presslabs.com/docs/code/${python.rootname}/
+
+checksums           rmd160  1bdfaec8284ac421fe6e78330ddc118748b599b4 \
+                    sha256  c0bdca63f3fc7cb88ab01d9a5cfd1b345de52ffe9c5cbcb1a27dbe03e0f66b49 \
+                    size    64165
+
+patchfiles          patch-mounter.py.diff \
+                    patch-utils-args.py.diff
+
+post-patch {
+    reinplace "s|@@PREFIX@@|${prefix}|g" ${worksrcpath}/gitfs/utils/args.py
+}
```

Comment:
At least this part needs to be moved into `if {${name} ne ${subport}} {…}` so that it isn't applied to the `py-gitfs` stub port.
```
--->  Patching args.py: s|@@PREFIX@@|/opt/local|g
DEBUG: Executing reinplace: /usr/bin/sed s|@@PREFIX@@|/opt/local|g </opt/local/var/macports/build/_Users_runner_work_1_s_python_py-gitfs/py-gitfs/work/gitfs-0.5.2/gitfs/utils/args.py >@file10
DEBUG: couldn't read file "/opt/local/var/macports/build/_Users_runner_work_1_s_python_py-gitfs/py-gitfs/work/gitfs-0.5.2/gitfs/utils/args.py": no such file or directory
    while executing
"exec -ignorestderr -- {*}$cmdline"
Error: reinplace: couldn't read file "/opt/local/var/macports/build/_Users_runner_work_1_s_python_py-gitfs/py-gitfs/work/gitfs-0.5.2/gitfs/utils/args.py": no such file or directory
Error: Failed to patch py-gitfs: reinplace sed(1) failed
```

Reply:
The post-activate and notes below also need to go into the existing `if {${name} ne ${subport}}` block.

Reply:
> … so that it isn't applied to the `py-gitfs` stub port.

Didn't think of that scenario. Thanks.

Done √